src/components/organisms/ActionsPane/ActionsPane.tsx

```diff
@@ -45,6 +45,16 @@ import {
 
 const {TabPane} = Tabs;
 
+const formSupportedResourceTypes = [
```

Comment:
rename to `formSupportedResourceKinds` ?

Reply:
Yeah - for sure